pyfar/dsp/dsp.py

```diff
@@ -1651,3 +1651,83 @@ def convolve(signal1, signal2, mode='full', method='overlap_add'):
 
     return pyfar.Signal(
         res, signal1.sampling_rate, domain='time', fft_norm=fft_norm)
+
+
+def decibel(signal, domain='freq', log_prefix=None, log_reference=1):
+    """Return array of given signal in decibel in choosen domain.
+
+    Parameters
+    ----------
+    signal : Signal, TimeData, FrequencyData
+        The signal which is converted into decibel
+    domain : str
+        Signal domain, in which the signal is converted into decibel
+        The different domains:
+
+        ``'freq'``
+            Converts signal in frequency domain. Signal must be of type
+            'Signal' or 'FrequencyData'.
+        ``'time'``
+            Converts signal in time domain. Signal must be of type
+            'Signal' or 'TimeData'.
+        ``'freq_raw'``
+            Converts signal in freq_raw domain. Signal must be of type
+            'Signal'.
+
+        The default is ``'freq'``.
+    log_prefix : int
+        Return prefix for dB calculation in frequency domain depending on
+        fft_norm..
+
+        For the FFT normalizations ``'psd'`` and ``'power'`` the prefix is 10,
+        for the other normalizations it is 20.
+
+        The default is None, but will be set during the fuction call.
```

Comment:
```suggestion
        The prefix for the dB calculation. The default ``None``, uses ``10``
        for signals with ``'psd'`` and ``'power'`` FFT normalization and
        ``20`` otherwise.
```